src/EnergyPlus/PluginManager.cc

```diff
@@ -55,41 +61,355 @@
 #include <EnergyPlus/PluginManager.hh>
 #include <EnergyPlus/UtilityRoutines.hh>
 
-#if LINK_WITH_PYTHON == 1
-#ifdef _DEBUG
-// We don't want to try to import a debug build of Python here
-// so if we are building a Debug build of the C++ code, we need
-// to undefine _DEBUG during the #include command for Python.h.
-// Otherwise it will fail
-#undef _DEBUG
-#include <Python.h>
-#define _DEBUG
-#else
-#include <Python.h>
-#endif
-#endif
-
 namespace EnergyPlus {
 namespace PluginManagement {
     std::unique_ptr<PluginManager> pluginManager;
 
-    std::map<int, std::vector<std::function<void ()> > > callbacks;
+    std::map<int, std::vector<std::function<void()>>> callbacks;
     std::vector<PluginInstance> plugins;
     std::vector<PluginTrendVariable> trends;
     std::vector<std::string> globalVariableNames;
     std::vector<Real64> globalVariableValues;
 
     // some flags
     bool fullyReady = false;
-    bool shouldIssueFatalAfterPluginCompletes = false;
+    bool apiErrorFlag = false;
+
+    // function pointers into the PythonWrapper DLL
+    void *wrapperDLLHandle = nullptr;
```

Comment:
These declarations store the handles to the new PythonWrapper DLL and the functions inside it.